index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles.css" rel="stylesheet" />
+    <link rel="shortcut icon" type="image/jpg" href="assets/filled-leaf-logo.png"/>
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/font-awesome/latest/css/font-awesome.min.css">
+    <title>mint | component library</title>
+</head>
+<body>
+    <header class="navigation">
+        <div class="nav-inner">
+            <div class="nav-logo">
+               <a href="index.html"><img src="assets/filled-leaf-logo.png" class="logo-nav"></a>
+               <a href="index.html" class="link-style"><h2>mint</h2>
+            </a>
+            </div>
+            <div class="nav-top hidden-sm">
+                <div class="nav-float-right"><a href = "index.html" class="hover-effect">home</a></div>

```

Comment:
Start name,,documentation and github in Caps. Although it's my personal opinion.